sdk/eventhub/Azure.Messaging.EventHubs/tests/Authorization/SharedAccessSignatureCredentialTests.cs

```diff
@@ -36,7 +36,7 @@ public void ConstructorValidatesTheSignature()
         public void ConstructorValidatesInitializesProperties()
         {
             var value = "TOkEn!";
-            var signature = new SharedAccessSignature(String.Empty, "keyName", "key", value, DateTime.UtcNow.AddHours(4));
+            var signature = new SharedAccessSignature(String.Empty, "keyName", "key", value, DateTimeOffset.Now.AddHours(4));
```

Comment:
These should be `DateTimeOffset.UtcNow.`    Not a huge deal in the tests, but `UtcNow` = the UTC offset (nothing) where `Now` uses the local timezone offset.  

[reference](https://docs.microsoft.com/en-us/dotnet/api/system.datetimeoffset.now?view=netframework-4.8)

Reply:
Done.